workers/src/main/java/edu/cornell/repository/JUnit5RepositoryImpl.java

```diff
@@ -1,27 +1,29 @@
 package edu.cornell.repository;
 
-import edu.cornell.TestOutputStream;
+import edu.cornell.testresultproducer.TestOutputStream;
+import edu.cornell.testresultproducer.TestOutputStream.TestResult;
 import java.io.File;
 import java.util.List;
-import lombok.Data;
 import lombok.EqualsAndHashCode;
 import lombok.NonNull;
-import lombok.RequiredArgsConstructor;
 import lombok.ToString;
-import lombok.extern.slf4j.Slf4j;
 
 /**
  * A class representing a Java project using JUnit 5 for testing
  */
-@Slf4j
 @EqualsAndHashCode(callSuper = true)
-@Data
+@ToString
 final class JUnit5RepositoryImpl extends Repository {
-    /** The root directory of the project once it's cloned */
-    private final @NonNull File rootDir;
+
+    JUnit5RepositoryImpl(@NonNull File rootDir) {
+        super(rootDir);
+    }
 
     @Override
-    public void test(@NonNull List<String> tests, TestOutputStream output) {
+    public void test(@NonNull List<String> tests, @NonNull TestOutputStream output) {
         // TODO: Owen
```

Comment:
is this TODO supposed to be left in?

Reply:
Should be removed once Owen merges his stuff. I'm going to keep it in just in case something goes wrong during the merge and I can remember where things need to be hooked up